ccd-definition/CaseEventToFields/CareSupervision/manageOrders/2_orderSelectionSection.json

```diff
@@ -36,6 +36,33 @@
     "PageID": "OrderSelection",
     "PageDisplayOrder": 2,
     "PageColumnNumber": 1,
+    "FieldShowCondition": "manageOrdersOperation = \"CREATE\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "manageOrdersUploadType",
+    "PageFieldDisplayOrder": 4,
+    "DisplayContext": "MANDATORY",
+    "PageID": "OrderSelection",
+    "PageDisplayOrder": 2,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "manageOrdersOperation = \"UPLOAD\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "manageOrdersUploadTypeOtherTitle",
+    "PageFieldDisplayOrder": 4,
+    "DisplayContext": "MANDATORY",
+    "PageID": "OrderSelection",
+    "PageDisplayOrder": 2,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "manageOrdersOperation = \"UPLOAD\" AND manageOrdersUploadType = \"OTHER_ORDER\"",
```

Comment:
could this condition be simplified to just `manageOrdersUploadType = \"OTHER_ORDER\"`, the first part of the condition is redundant as `manageOrdersUploadType` won't have a value unless upload is selected

Reply:
Unfortunately using the preview button that will show if change the type